qcodes/__init__.py

```diff
@@ -32,6 +32,7 @@
 from qcodes.loops import Loop, active_loop, active_data_set
 from qcodes.measure import Measure
 from qcodes.actions import Task, Wait, BreakIf
+from qcodes.monitor.monitor import Monitor
```

Comment:
We might want to drop this import to not make websockets a hard dependency of qcodes?